packages/nodejs/src/span.ts

```diff
@@ -17,6 +17,11 @@ import { BaseClient } from "./client"
  */
 export class BaseSpan implements Span {
   protected _ref: unknown
+  #_open: boolean
```

Comment:
Nitpick: the `#` already means "private" -- I'd drop the `_`, which also means "private".

Reply:
Good catch!